diff --git a/dist/index.esm.js b/dist/index.esm.js
index 5c9820b1a261305d500e69eeaf350821872beaf4..db803c7d59c0951198157c0e6c46aad3af3bfadc 100644
--- a/dist/index.esm.js
+++ b/dist/index.esm.js
@@ -62866,7 +62866,7 @@ function rangeValueToHtml(ctx, sheetId, ranges) {
         if (_.isNil(c_value)) {
           c_value = "";
         }
-        column += escapeHTMLTag(c_value);
+        column += _.escape(c_value);
       } else {
         var _style = "";
         if (borderInfoCompute && borderInfoCompute["".concat(r, "_").concat(c)]) {
diff --git a/dist/index.js b/dist/index.js
index 04113a18dee9feee28e9434ee3023626c800e7d3..8304933d13c14e87d8f0c4e84d0a7a02546b0b5b 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -62876,7 +62876,7 @@ function rangeValueToHtml(ctx, sheetId, ranges) {
         if (___default['default'].isNil(c_value)) {
           c_value = "";
         }
-        column += escapeHTMLTag(c_value);
+        column += ___default['default'].escape(c_value);
       } else {
         var _style = "";
         if (borderInfoCompute && borderInfoCompute["".concat(r, "_").concat(c)]) {
